honeybee_model_schema/energy/programtype.py

```diff
@@ -37,19 +37,14 @@ def check_name(cls, v):
         'value is 0.30.'
     )
 
-    latent_fraction: Union[float, str] = Schema(
-        'autocalculate',
+    latent_fraction: float = Schema(
+        default=None,
         ge=0,
         le=1,
-        description='Used to specify a fixed latent fraction of heat gain due to people.'
+        description='Used to specify a fixed latent fraction of heat gain due to people. \
+            If left null then Honeybee will autocalculate this value.'
```

Comment:
This seems less explicit to me and it goes against a convention of spelling out the property that I know Mostapha originally wanted to follow.. What do you think about this @mostapharoudsari ?

Reply:
Agreed it's less explicit. The other choice we have (i think) if we want to be more explicit and keep things strongly typed for the benefit of JSON schema and strongly typed languages like C# is to use a nested object inside as proposed for the ideal air limits.